python/setup.py

```diff
@@ -235,6 +235,9 @@ def get_packages(self):
         "adag": [
             "cupy-cuda12x; sys_platform != 'darwin'",
         ],
+        "cg": [
+            "cupy-cuda12x; sys_platform != 'darwin'",
+        ],
```

Comment:
could you add a comment saying `adag` is just an alias of `cg`?

Reply:
better if you just remove `adag` here, and have an `setup_spec.extras["adag"] = list(setup_spec.extras["cg"]` below